.travis.yml

```diff
@@ -63,6 +76,9 @@ jobs:
     language: shell
     install:
     script: ./scripts/build_windows.cmd
+    env:
+      - PYTHON_VERSION=3.7.5
+      - PATH=/c/Python37:/c/Python37/Scripts:$PATH
```

Comment:
Does windows pkgs have python interpreter inside?

Reply:
PyInstaller does not support `3.8`. Ref: https://github.com/pyinstaller/pyinstaller/issues/4311